src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyAloneOrSingleline.java

```diff
@@ -209,4 +209,18 @@ public void codeAfterLastRightCurly() {
         }; // violation
         for (int i = 0; i < 1; i++) { new Object(); }; // violation
     }
+
+    enum TestEnum {}
+
+    enum TestEnum1 {
+        SOME_VALUE;} //violation
+
+    enum TestEnum2 { SOME_VALUE; }
+
+    enum TestEnum3 {
+        SOME_VALUE;
+    }
+
+    enum TestEnum4 { SOME_VALUE; }
```

Comment:
Looking at other Enum4, it looks like right curly should be on the next line.

Reply:
@rnveach I have made the changes.